definitions/marts/looker_studio_marts/ecf/ls_ecf_reg_ect_inductions.sqlx

```diff
@@ -0,0 +1,177 @@
+        school_transfer: "TRUE if a participant has transferred out of/in to the school in that induction record.",
+        previous_school_id: "School ID from the preceding induction record for the participant",
+        induction_status: "A status value used to manage the state of an induction record. It can indicate that a record is the active record or that something has changed and the record is not the active record (or will not be in the future). Potential values: active, changed, leaving, withdrawn ",
+        training_status: "This is managed on the lead provider side and indicates whether the participant is being trained by a provider or not. Potential Values: active, deferred, withdrawn",
+        completion_date: "The date an ECT completed their induction.",
+        reg_year: "The registration year of the induction record",
+        ect_21: "TRUE/FALSE flag that checks if this ECT was rolled over from 2021 to 2024",
+        mentor_cohort: "The cohort of the associated mentor of the ECT",
+        mentor_completion_date: "The date the corresponding mentor of the ECT completed their training",
+        mentor_21: "TRUE/FALSE flag that checks if the corresponding mentor was rolled over from 2021 to 2024",
+        min_created_record: "TRUE/FALSE flag that checks if this induction record was the first created for the participant + school combination in a given registration year",
+        max_created_record: "TRUE/FALSE flag that checks if this induction record was the last created for the participant + school combination in a given registration year",
+        transfer_in: "TRUE/FALSE flag that checks if the induction record has a different school_id than the prior induction record and the record is flagged as a transfer record, identifying this participant having transferred to a new school",
+        matched_cohort_reg_year: "TRUE/FALSE flag that checks if the registration year of the induction record matches the cohort of the participant confirming this is a new participant in this year",
+        assigned_mentor_status: "Checks if a mentor has been assigned to the participant",
+        mentor_type: "Classification of the type of mentor assigned to the ECT, checking where the mentor is in their training and which cohort they come from. Values are either: No Mentor Assigned, Completed Mentor, Mentor training in prior cohort, Mentor from same cohort",
+        adjusted_creation_date: "This shifts the creation date of the induction record forward to the beginning of registration of a given year: 2022 - 10/05/2022, 2023 - 05/06/2023, 2024 - 17/06/2024"
+    }
+}
+
+WITH
+  -- #This table generates a list of the dates from May - October in each approximate registration window (May - October) #
+  registration_dates AS (
+SELECT
+  *
+FROM (
+  SELECT
+    DISTINCT(start_year) AS reg_year,
+    CAST(CONCAT(start_year,'-05-01') AS date) AS reg_start_date,
+    CAST(CONCAT(start_year,'-10-31') AS date) AS reg_end_date
+  FROM

```

Comment:
The FROM clause is not using the Javascript `${ref()}`

Reply:
Sorry about that! Have added the Javascript across all marts.